bin/build.js

```diff
@@ -0,0 +1,19 @@
+const fs = require( 'fs' ),
+	path = require( 'path' );
+
+( async () => {
+	var rd = fs.createReadStream( path.resolve( path.join( __dirname, '..', 'assets', 'dev', 'licenses.json' ) ) );
```

Comment:
`var` => `const`. Also, can this variable to be renamed to `read`? The same for the `wr` one. At first it's not clear what this means. Code should be readable imo :wink: